### PR TITLE
Feature/create feed page

### PR DESCRIPTION
**Pull request by nickolay-grechkin**

### CODE REVIEW

File: backend/core/app/src/main/resources/application.properties [hunk 1]
```diff
@@ -6,8 +6,13 @@ spring.servlet.multipart.max-request-size=-1
 # Database PostgresSQL
 #-------------------------
 spring.datasource.driver-class-name=org.postgresql.Driver
+<<<<<<< Updated upstream
 spring.datasource.username=${DB_USER}
 spring.datasource.password=${DB_PWD}
+=======
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+>>>>>>> Stashed changes
```
Comment (kopot-o-d) [inline backend/core/app/src/main/resources/application.properties:15]: seems like u didn't resolve all merge conflicts

File: backend/core/app/src/main/resources/application.properties [hunk 2]
```diff
@@ -6,6 +6,7 @@ spring.servlet.multipart.max-request-size=-1
 # Database PostgresSQL
 #-------------------------
 spring.datasource.driver-class-name=org.postgresql.Driver
+<<<<<<< Updated upstream
```
Comment (kopot-o-d) [inline backend/core/app/src/main/resources/application.properties:9]: remove

**Review by kopot-o-d — COMMENTED** [review]
move ALL SVGs to separate components

**Review by Quipex — CHANGES_REQUESTED** [review]
Merge with the latest dev to trigger auto-linter

File: frontend/src/screens/FeedPage/routines/index.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable max-len */
+import { createRoutine } from 'redux-saga-routines';
+
+const createFeedPageRoutine = <T extends unknown>(actionName: string) => createRoutine<T>(`FEED_PAGE:${actionName}`);
```
[the remark this answers is not in the document]
Comment (nickolay-grechkin) [reply frontend/src/screens/FeedPage/routines/index.ts:4]: yes

**Review by kopot-o-d — APPROVED** [review]

**Review by kopot-o-d — APPROVED** [review]

**Pull request merged**